src/test/localisation.test.ts

```diff
@@ -1,10 +1,16 @@
-import { LocalesService } from "@companieshouse/ch-node-utils";
+import { NextFunction, Request, Response } from "express";
 import request from "supertest";
+import { LocalesService } from "@companieshouse/ch-node-utils";
+
 import app from "../app";
 import * as config from "../config";
 import enStartPageText from "../../locales/en/translations.json";
 import cyStartPageText from "../../locales/cy/translations.json";
 
+jest.mock("../middlewares/authentication.middleware", () => ({
```

Comment:
i think the authentication mock is going to be needed for a lot of tests, could it be a candidate to be moved into the setup-jest.ts file that runs before each test?  If a test does not need to mock it then I think we can use jest.requireActual to not use the mocked version.

Reply:
I will move it

Reply:
move to setup